src/main/resources/reference.conf

```diff
@@ -8,6 +8,7 @@ conseil: {
   port: 1337
   cache-ttl: 15 minutes
   max-query-result-size: 100000
+  high-cardinality-limit: 100
```

Comment:
If this is missing from the config file, how will the application react?

Reply:
conseil will not start because of missing value in the config